### DISCUSSION TITLE
set up some chat room for tinyproxy technical conversations?

### DISCUSSION THREAD

**Discussion by obnoxxx** [tinyproxy/tinyproxy#473] [Ideas]
Hi, @rofl0r - I have not been usin IRC any more since quite a while but  I would appreciate having the opportunity to chat about tinyproxy directly or /and in a dedicated forum/channel .... any opinion about getting set up with some chat service? slack seems to be rather popular these days. or something matrix / element based? Any thoughts / preferences?

**Comment by rofl0r** [comment]
hi @obnoxxx - any specific reason you're avoiding IRC ? i personally am kinda reluctant to use proprietary websites for discussions, especially as one has to keep those tabs activate and they're using some js in the background that constantly produces CPU load. i guess matrix would be ok as it can be bridged to IRC...

**Reply by obnoxxx** [reply]
@rofl0r  - No specific reason to avoid IRC. I just don't have a system any more to access it. I can try again.

yeah, matrix has an IRC bridge that can work well enough I guess, and it does have the mobile element client, so you don't have to use it in a browser ...
